validation/standalone-constraint-constraint.ttl

```diff
@@ -17,6 +17,12 @@
 <ObservationConstraintShape>
     a sh:NodeShape ;
     sh:targetClass cube:Constraint ;
+    sh:property [
+        # a cube constraint shall constrain its observations
+        sh:path sh:targetClass ;
+        sh:hasValue cube:Observation;
+        sh:message "cube:Constraint needs a sh:targetClass cube:Observation"
+    ] ;
```

Comment:
Every cube's observations have that class. When we start adding `sh:targetClass cube:Constraint` to cube shapes these shapes will not only constrain their own observations but ALL observations across all cubes.

See [here](https://s.zazuko.com/2tWqa9v) how the dimensions are constrained across cubes

Reply:
@tpluscode 💯 true
@kronmar can you jump in an describe your observed issue more precisely ... THX